raiden/transfer/state.py

```diff
@@ -151,7 +151,7 @@ def __eq__(self, other: Any) -> bool:
         )
 
 
-@dataclass(eq=True)
+@dataclass(eq=True, order=True)
```

Comment:
Why do we need this?

Reply:
this adds comparison methods on `Claim` and so allows us to call `sorted([Claim(...), Claim(...)])`.

Since it does a field by field comparison in the order of definition in the dataclass, this allows to quickly compare two claims and find the one with the higher amount (all other fields being equal)